src/components/common/ga4-opl/template.njk

```diff
@@ -0,0 +1,17 @@
+<script {% if params.nonce%} nonce='{{ params.nonce }}'{%  endif %}>
```

Comment:
You added twice the same nunjuck component. Into src/components/common and src/components . You have to delete 1 of them.